website/utils/erv_status/erv_status.js

```diff
@@ -0,0 +1,50 @@
+/***
+ * Create the ERV status warning UI element
+ */
+function createERVStatus(toHideStatus,img_path, description)
+{
+    let divERVStatusWrapper = document.createElement('div');
+    divERVStatusWrapper.id='erv_status_wrapper';
+    if(toHideStatus) 
+        divERVStatusWrapper.style.display='none';
+
+    let p_erv_status_div = document.createElement('div');
+    p_erv_status_div.id="erv_status_description_div";
+    //Lane Change description
```

Comment:
Looks like a carry-over comment from `lane_change_status.js`, can we update it for this new functionality?

Reply:
Addressed https://github.com/usdot-fhwa-stol/carma-web-ui/pull/160/commits/52494d9fb3ec417a41e3624e83bb26dbf1159852